### DISCUSSION TITLE
Question about NextAuth.js + Electron

### DISCUSSION THREAD

**Discussion by itzheropvp** [nextauthjs/next-auth#6445] [Help]
### Question 💬

Hi, i'm using for the first time NextAuth.js because i need to create a login page with Discord and check if has roles in a server an access to the dashboard.

It's an app created with Nextron (Nexjs + Electron). 
When i start my app i noticed that if i use the function `signIn('discord')` it redirects me to a blank page forever until i close the app.

I searched online and i saw that the "solution" was to setting `nodeIntegration: false`, doing so all my pages where white and the console says 'global is not defined'. 

I don't know how to fix it or other.

This is what happens: `(nodeIntegration: true)`
![image](https://user-images.githubusercontent.com/51835843/213650406-9d49b8d3-ee47-4ec3-be38-938ef0eb03a1.png)

With `(nodeIntegration: false)`:
![image](https://user-images.githubusercontent.com/51835843/213650792-c2d309a0-0c01-4593-a61c-625ee03ecfc6.png)

### How to reproduce ☕️

```typescript
const browserOptions: BrowserWindowConstructorOptions = {
    ...state,
    ...options,
    webPreferences: {
      nodeIntegration: true, // What i changed, from true to false.
      contextIsolation: false,
      ...options.webPreferences,
    },
  };
  win = new BrowserWindow(browserOptions);
  win.on('close', saveState);
```

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by Mogeiman** [comment]
Hey did you ever get to integrate nextauth with nextron, cus I am currently struggling with it. I keep on getting require is not defined in my code, even though I am not using require anywhere in my code. This only happens when I wrap SessionProvider from nextauth around my app. Help?

**Comment by s3723766** [comment]
I am also curious if you got the next-auth functionality working in nextron? 
I currently have a next.js application using next-auth deployed via Vercel at let's say demo.com, I want to use the existing backend api at demo.com/api and have the nextron application interface with it (both secured and unsecured routes). 
Does anyone have any idea's?

**Reply by niudai** [reply]
I have the same application, looking to a solution now

**Reply by cammycurry** [reply]
Ever find a solution?

**Comment by cammycurry** [comment]
Any luck anywhere?

**Comment by MaX-NeO** [comment]
any updates ?
